pyspark/bigdl/keras/converter.py

```diff
@@ -1614,7 +1621,7 @@ def create_highway(self):
         else:
             activation = get_activation_by_name(self.config["activation"],
                                                 "%s_%s" % (self.config["name"], self.config["activation"]))
-        blayer = BLayer.Highway(size=self.input_shape[1],
+        blayer = BLayer.Highway(size=int(self.input_shape[1]),
```

Comment:
is this a must that surround with int?

Reply:
I don't know. But you surround everything related to shape with int and thus I do this for everything to be consistent.